pkg/globalnet/controllers/ipam/globalnet.go

```diff
@@ -135,6 +136,14 @@ func (i *Controller) evaluateService(service *k8sv1.Service) Operation {
 		return Ignore
 	}
 
+	_, err := i.svcExGetter(service.Namespace, service.Name)
+	if errors.IsNotFound(err) {
+		return Ignore
+	} else if err != nil {
+		klog.Errorf("Failed to get ServiceExport for %#v due to %v", service, err)
+		return Requeue
```

Comment:
I think we should simply return Ignore here if ServiceExport object is missing. For example, say the user created a regular ClusterIPService and is not interested in exporting it, so we will be requeuing it again and again. Am I missing something?

Reply:
We return `Ignore` for `IsNotFound` above.

Reply:
Okay, that makes sense. I missed the IsNotFound() check.